src/main/java/org/zone/commands/structure/ZoneCommands.java

```diff
@@ -47,7 +48,7 @@
     ZoneFlagBlockPlaceSetGroupCommand ZONE_FLAG_BLOCK_PLACE_SET_GROUP_COMMAND = new ZoneFlagBlockPlaceSetGroupCommand();
     ZoneViewBalanceCommand ZONE_FLAG_VIEW_BALANCE_COMMAND = new ZoneViewBalanceCommand();
     SetGreetingsMessageCommand SET_GREETINGS_MESSAGE_COMMAND = new SetGreetingsMessageCommand();
-
+    GreetingsMessageViewCommand GREETINGS_MESSAGE_VIEW_COMMAND = new GreetingsMessageViewCommand();
```

Comment:
Could the class be renamed to ``ZoneFlagGreetingsViewCommand`` to make it consistant with the others

Reply:
DONE will update